integration-commons/src/test/java/org/symphonyoss/integration/exception/authentication/UnexpectedAuthExceptionTest.java

```diff
@@ -0,0 +1,41 @@
+package org.symphonyoss.integration.exception.authentication;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Unit tests for {@link UnexpectedAuthException}.
+ * Created by crepache on 23/06/17.
+ */
+public class UnexpectedAuthExceptionTest {
+
+  private String message = "message";
+
+  @Test
+  public void testUnexpectedAutoException() {
+    UnexpectedAuthException exception = new UnexpectedAuthException(message);
+    String resultMessage = exception.getMessage();
+    String expectedMessage = "\n" +
```

Comment:
You could create a private method to build these String messages and avoid repeated code.